nelc/brand-openedx
# feat: set default typography

## Description
This sets the default font for NELP, it was not possible to use the font file since any change of that file is not being applied to application [issue](https://edunext.atlassian.net/browse/FUTUREX-968) 

## How to test with app learning
1. Clone paragon and brand-openedx repo into the src/mfe folder (with right versions paragon=V23.0.0-alpha.1 from openedx repo  brand-openedx=open-release/redwood.nelp from nelc repo)
2. Add the module.config.js file into the frontend-app-learning folder
```js
module.exports = {
    localModules: [
        { moduleName: '@edx/brand', dir: '../brand-openedx'},
        { moduleName: '@edx/paragon/styles/css/themes', dir: '../paragon', dist: 'styles/css/themes' },
        { moduleName: '@edx/paragon/styles/css/core', dir: '../paragon', dist: 'styles/css/core' },
        { moduleName: '@edx/paragon/styles/scss/core', dir: '../paragon', dist: 'styles/scss/core' },
        { moduleName: '@edx/paragon/icons', dir: '../paragon', dist: 'icons' },
        { moduleName: '@edx/paragon', dir: '../paragon', dist: 'src' },
    ],
};
```
3. Restart the learning mfe service, docker attach container and then ctrl+c
4. checkout this branch
5. if you have set `PARAGON_THEME_URLS` delete it from the `MFE_CONFIG`
6. Go to any course page and check the font

## Before
![image](https://github.com/user-attachments/assets/0e703867-5ab3-48ce-995c-df866a6c358a)

## After
![image](https://github.com/user-attachments/assets/90fc25b9-fdf2-4471-add1-eb8665bdbbaf)
